### ISSUE TITLE
Items sometimes disappear with warning in log

### ISSUE THREAD

**Issue by TBlueF** [SpongePowered/Sponge#1434]
**Minecraft:** 1.11.2
**SpongeAPI:** 6.1.0-SNAPSHOT-c4a7afd
**SpongeForge:** 1.11.2-2393-6.1.0-BETA-2442
**Minecraft Forge:** 13.20.1.2393

On my server, sometimes items disappear after beeing thrown and collected again in the players inventory.
This warning seems to appears in the fml-server-latest.log almost simultaneously and every time this happens:
```
[22:04:12] [Server thread/WARN] [Sponge/]: Unable to find InventoryPacketState handler for click window packet: CPacketClickWindow{windowId=26, slotId=-999, usedButton=2, actionNumber=2, clickedItem=1xtile.air@0, mode=CLONE}
```
*(Full log: https://paste.craftednature.de/fml-server-latest.log.report1.txt)*

I sadly can't reproduce this consistently, so i wasn't able to test this further. Maybe that warning isn't even related to the dissappearing items? I hope you can tell me more :)

**Comment by XakepSDK** [comment]
https://github.com/SpongePowered/SpongeCommon/issues/1402 ?

**Comment by Zidane** [comment]
@Faithcaio Thoughts?

**Comment by Faithcaio** [comment]
Can't reproduce.
Might have been fixed with the same commit as in #1750.
No idea how items would disappear with it.
